runtime/src/serde_snapshot.rs

```diff
@@ -173,6 +173,57 @@ where
     })
 }
 
+#[allow(clippy::too_many_arguments)]
+pub(crate) fn bank_from_stream_incremental<R>(
+    serde_style: SerdeStyle,
+    fss_stream: &mut BufReader<R>,
+    iss_stream: &mut BufReader<R>,
+    account_paths: &[PathBuf],
+    unpacked_append_vec_map: UnpackedAppendVecMap,
+    genesis_config: &GenesisConfig,
+    frozen_account_pubkeys: &[Pubkey],
+    debug_keys: Option<Arc<HashSet<Pubkey>>>,
+    additional_builtins: Option<&Builtins>,
+    account_indexes: AccountSecondaryIndexes,
```

Comment:
nit: might also be a good time to rename `account_indexes` -> `account_secondary_indexes`

Reply:
I agree. Will do.

Reply:
Done!